dvm.sh

```diff
@@ -27,7 +27,11 @@ fi
 if [ -n "$BASH_SOURCE" ]; then
   DVM_SCRIPT_SOURCE="${BASH_SOURCE[0]}"
 fi
-export DVM_HELPER="$(command cd $DVM_CD_FLAGS "$(dirname "${DVM_SCRIPT_SOURCE:-$0}")" > /dev/null && command pwd)/dvm-helper/dvm-helper"
+if command -v builtin >/dev/null 2>&1; then
+  export DVM_HELPER="$(builtin cd $DVM_CD_FLAGS "$(dirname "${DVM_SCRIPT_SOURCE:-$0}")" > /dev/null && command pwd)/dvm-helper/dvm-helper"
+else
+  export DVM_HELPER="$(cd $DVM_CD_FLAGS "$(dirname "${DVM_SCRIPT_SOURCE:-$0}")" > /dev/null && command pwd)/dvm-helper/dvm-helper"
```

Comment:
Is there a reason why `pwd` (and `rm` later down in the file) aren't also getting the same treatment as `cd`?

If we need it in both plances, perhaps we can have a variable `$CMD_PREFIX` or something like that which is set to either `builtin` or `""`?

Reply:
The reason why the `pwd` and `rm` programs aren't being invoked by the `builtin` is because the **OSX** doesn't have a specific **built-in** command for them, like it have for `cd`.